### PR TITLE
Update to browser@fcc7f9e5f4

### PR DESCRIPTION
**Pull request by inancgumus**
## What?

Updates k6-core for the v0.50 canary release.

Just putting it here to make a release.

## Checklist

- [x] I have performed a self-review of my code.
- [ ] I have added tests for my changes.
- [x] I have run linter locally (`make lint`) and all checks pass.
- [x] I have run tests locally (`make tests`) and all tests pass.
- [ ] I have commented on my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# v1.55.2
+# v1.56.2
```
Comment (mstoykov) [inline .golangci.yml:1]: I think this shouldn't be in this PR ;)

**Review by mstoykov — DISMISSED** [review]
LGTM apart from the golangci-lint bump

File: .golangci.yml (hunk 1 above)
Comment (inancgumus) [reply .golangci.yml:1]: Yeah, agree. But the linter is failing :(

Comment (inancgumus) [reply .golangci.yml]: Removed the bumping commit.

Comment (inancgumus) [reply .golangci.yml]: Now it works, weird.

**Review by mstoykov — APPROVED** [review]

**Comment by mstoykov** [comment]
Closing this in favor of #3625

**Pull request closed without merging**